Is this MCP project functioning

I have cloned it and built it according the instructions, but then I get an error when trying to connect to it from Cline.

The error looks like this:

```
node:internal/modules/esm/resolve:265 throw new ERR_MODULE_NOT_FOUND( ^ Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/username/repos/mcp-github-project-manager/build/infrastructure/github/GitHubRepositoryFactory' imported from /home/username/repos/mcp-github-project-manager/build/services/ProjectManagementService.js at finalizeResolution (node:internal/modules/esm/resolve:265:11) at moduleResolve (node:internal/modules/esm/resolve:933:10) at defaultResolve (node:internal/modules/esm/resolve:1169:11) at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:542:12) at ModuleLoader.resolve (node:internal/modules/esm/loader:510:25) at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:239:38) at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:96:40) at link (node:internal/modules/esm/module_job:95:36) { code: 'ERR_MODULE_NOT_FOUND', url: 'file:///home/username/repos/mcp-github-project-manager/build/infrastructure/github/GitHubRepositoryFactory' } Node.js v20.18.1 MCP error -1: Connection closed
```

Is that just a configuration problem?

--- a/src/index.ts
+++ b/src/index.ts
@@ -7,7 +7,7 @@ import {
   ListToolsRequestSchema,
   McpError,
 } from "@modelcontextprotocol/sdk/types.js";
-import { ProjectManagementService } from "./services/ProjectManagementService";
+import { ProjectManagementService } from "./services/ProjectManagementService.js";
 
 function getRequiredEnvVar(name: string): string {
   const value = process.env[name];

--- a/src/infrastructure/github/GitHubRepositoryFactory.ts
+++ b/src/infrastructure/github/GitHubRepositoryFactory.ts
@@ -1,8 +1,8 @@
-import { GitHubConfig } from "./GitHubConfig";
-import { GitHubIssueRepository } from "./repositories/GitHubIssueRepository";
-import { GitHubMilestoneRepository } from "./repositories/GitHubMilestoneRepository";
-import { GitHubProjectRepository } from "./repositories/GitHubProjectRepository";
-import { GitHubSprintRepository } from "./repositories/GitHubSprintRepository";
+import { GitHubConfig } from "./GitHubConfig.js";
+import { GitHubIssueRepository } from "./repositories/GitHubIssueRepository.js";
+import { GitHubMilestoneRepository } from "./repositories/GitHubMilestoneRepository.js";
+import { GitHubProjectRepository } from "./repositories/GitHubProjectRepository.js";
+import { GitHubSprintRepository } from "./repositories/GitHubSprintRepository.js";
 
 export class GitHubRepositoryFactory {
   private static instance: GitHubRepositoryFactory;

--- a/src/services/ProjectManagementService.ts
+++ b/src/services/ProjectManagementService.ts
@@ -7,7 +7,7 @@ import {
   Sprint,
   SprintId,
 } from "../domain/types";
-import { GitHubRepositoryFactory } from "../infrastructure/github/GitHubRepositoryFactory";
+import { GitHubRepositoryFactory } from "../infrastructure/github/GitHubRepositoryFactory.js";
 
 export class ProjectManagementService {
   private factory: GitHubRepositoryFactory;

